.github/scripts/generate-crd-docs/generate-crd-docs.sh

```diff
@@ -45,13 +46,18 @@ for api_group in "$OPERATOR_API_ROOT"*; do
 
     OUTPUT_PATH="./docs/content/en/docs/crd-ref/$sanitized_api_group/$sanitized_api_version"
 
+    renderer_config_file=$RENDERER_CONFIG_FILE
```

Comment:
maybe it is worth to make this flexible for future extra changes 🤔 for instance when we move towards beta in metrics op, we could put the default file and look if a file with the sanitizedversionname exists and in that case use that